### PR TITLE
[ci-visibility] Fix `cypress open` error when passing `experimentalInteractiveRunEvents`

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
Fix crash when using `cypress open` (interactive mode).

### Motivation
Fix customer issue. 

### Plugin Checklist
No tests, see additional notes.

### Additional Notes
⚠️ **Important**: no automated tests are possible in this case, since the error only happens with using the interactive mode `cypress open`, and there's no way to automate this. 

### Security 
Datadog employees:
- [ ] If this PR touches code that signs or publishes builds or packages, or handles credentials of any kind, I've requested a review from `@DataDog/security-design-and-guidance`.
- [x] This PR doesn't touch any of that.

Unsure? Have a question? Request a review!

### CODE REVIEW

File: integration-tests/cypress/cypress.spec.js [hunk 1]
```diff
@@ -55,7 +54,7 @@ moduleType.forEach(({
   testCommand
 }) => {
   // cypress only supports esm on versions >= 10.0.0
-  if (type === 'esm' && semver.satisfies(version, '<10.0.0')) {
+  if (type === 'esm' && version === '6.7.0') {
```
Comment (juan-fernandez) [inline integration-tests/cypress/cypress.spec.js:57]: unrelated fix: `version` will either be `6.7.0` or `latest`. `semver` should not work with `latest`

**Review by anmarchenko — APPROVED** [review]
